File: tournaments/myReverse/myReverse.js
function myReverse(input) {

    for (var i = 0; i * 2 < input.length; i++) {
        var tmp = input[i];
        input[i] = input[input.length - i - 1];
        input[input.length - i - 1] = tmp;
    }
    
    return input;
 }
  

File: tournaments/myReverse/myReverse.cpp
std::vector<int> myReverse(std::vector<int> input) {

   for (int i = 0; i * 2 < input.size(); i++) {
     int tmp = input[i];
     input[i] = input[(int)input.size() - i - 1];
     input[(int)input.size() - i - 1] = tmp;
   }
   return input;
}
